scripts/release/bump_versions.py

```diff
@@ -0,0 +1,225 @@
+"""Bump versions in files for release.
```

Comment:
How is this different from https://github.com/osohq/oso/blob/f89ae9d95678b68697c6dbfc736ac14ee89b09b8/scripts/bump_versions.py?

Reply:
same

Reply:
rm one of the two & 🚢 ?